modules/standard/CommDiagnostics.chpl

```diff
@@ -351,9 +351,9 @@ module CommDiagnostics
       D(loc.id).test_nb = chpl_numCommTestNB();
       D(loc.id).wait_nb = chpl_numCommWaitNB();
       D(loc.id).try_nb = chpl_numCommTryNB();
-      D(loc.id).fork = chpl_numCommForks();
-      D(loc.id).fork_fast = chpl_numCommFastForks();
-      D(loc.id).fork_nb = chpl_numCommNBForks();
+      D(loc.id).execute_on = chpl_numCommForks();
```

Comment:
If there is a very good reason, yes it can wait. But I'd rather have it done now because (1) this is the best time for that change since otherwise all we have done is solved part of the problem and moved the naming mismatch from one place to another, and (2) in recent experience, such "future" work never quite gets gotten to.